utils/gen-unicode-data/Sources/GenScalarProps/BinProps.swift

```diff
@@ -0,0 +1,289 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2021 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import GenUtils
+
+struct BinProps: OptionSet {
+  var rawValue: UInt64
+  
+  static let changesWhenCaseFolded       = BinProps(rawValue: 1 << 0)
```

Comment:
This is just a script so it's not super important, but computed vars are typically better as they don't require (even static) storage.